pantheon/src/test/java/tech/pegasys/pantheon/cli/PantheonCommandTest.java

```diff
@@ -1343,6 +1343,28 @@ public void parsesInvalidFastSyncMinPeersOptionWrongFormatShouldFail() {
         .contains("Invalid value for option '--fast-sync-min-peers': 'ten' is not an int");
   }
 
+  @Test
+  public void parsesValidFastSyncTaskCacheSizeOption() {
```

Comment:
If we move this flag to the experimental style in `SychronizerOptions` as suggested above, you should be able to cut these tests and just add a call to customize the cache size options here: https://github.com/PegaSysEng/pantheon/blob/12d19a152d238ea5629e20997963865aa4784ee2/pantheon/src/test/java/tech/pegasys/pantheon/cli/options/SynchronizerOptionsTest.java#L31-L61